local_groups/forms.py

```diff
@@ -95,14 +95,14 @@ class GroupLeaderSyncForm(forms.Form):
 
 
 # Customize AuthenticationForm as needed
-class GroupLoginForm(AuthenticationForm):
+class OrganizingHubLoginForm(AuthenticationForm):
```

Comment:
Can we move this to organizing hub app too? That's really where it belongs anyway, and otherwise this name doesnt make sense inside local groups app.

Reply:
👍